build.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
 
+# needed to build multi-platform images
+docker buildx create --name roboflow
+docker buildx use roboflow
+
 echo "🛠 : Building roboflow/inference-server:cpu"
-docker buildx build --platform linux/amd64 -t "roboflow/inference-server:cpu" -f- . < Dockerfile.cpu
```

Comment:
Nice! So does this auto pull the correct arch after you push it up?

Reply:
Yeah, it does!